pyworkflow/protocol/protocol.py

```diff
@@ -1982,7 +1982,7 @@ def getPackageCitations(self, bibTexOutput=False):
 
     def citations(self):
         """ Return a citation message to provide some information to users. """
-        citations = self.getCitations().values()
+        citations = list(self.getCitations().values())
```

Comment:
This partially fixes #24

Reply:
Partially? For me it works.

Reply:
It opens the window but the articles are just lists not links, also open bibtex doesn't work. But I have to double check

Reply:
Both cases work for me.